ios/Plugin/Plugin.swift

```diff
@@ -35,6 +35,15 @@ public class NativeBiometric: CAPPlugin {
         obj["isAvailable"] = false
         obj["biometryType"] = 0
         
+        if LAContext.biometricsChanged() {
+            obj["isBiometryChanged"] = true
+        }else {
+            // Handle biometrics changed
+            obj["isBiometryChanged"] = false
+            // Reset LAContext.savedBiometricsPolicyState to nil after doing so
+            LAContext.savedBiometricsPolicyState = nil
+        }
+
```

Comment:
```
obj["isBiometryChanged"] = LAContext.biometricsChanged()

if !obj["isBiometryChanged"] {
  // Reset LAContext.savedBiometricsPolicyState to nil after doing so
  LAContext.savedBiometricsPolicyState = nil
}
```
Also question, is this expected to nill the `LAContext.savedBiometricsPolicyState` everytime we detect no changes?

Reply:
yes to clear the biometry data

Reply:
Hmmm, so if biometric is not changed (`isBiometryChanged = false`) we clear biometric data? :thinking:

Reply:
oh sorry it's upside down, thanks for checking